.github/workflows/pre-commit.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        k8s: [v1.29.1, v1.28.6, v1.27.10, v1.26.13, v1.25.16, v1.24.17]
+        k8s: [v1.29.2, v1.28.7, v1.27.11, v1.26.14, v1.25.16, v1.24.17]
```

Comment:
Nit: consider rename the JIRA ticket (and the PR/commit) to be more accurate, ie. this change also changes the test matrix.

Reply:
Weird, the changes you mentioned seems to be already merge at #833, commit hash also same as #833. Will further investigate that!